docs/classic-ui/images.md

```diff
@@ -0,0 +1,226 @@
+---
+html_meta:
+  "description": "Image resolution and scaling in Plone Classic UI"
+  "property=og:description": "Image resolution and scaling in Plone Classic UI"
+  "property=og:title": "Image resolving and scaling"
+  "keywords": "Plone, Classic UI, classic-ui, image, resize, scale"
+---
+(classic-ui-images-label)=
+
+# Image handling
+
+The default content types and behaviors are using a `plone.namedfile.NamedBlobImage` field for all images.
+
+For this fields, image scaling and HTML tag creation is provided by the `plone.namedfile.scaling` module, specifically in the [`ImageScaling`](https://github.com/plone/plone.namedfile/blob/ecf33a2bc7a8c61888909bc383b3e08d80888e43/plone/namedfile/scaling.py#L350) view.
+
+Given a Dexterity content type named `news-item1` with a `plone.namedfile.NamedBlobImage` field named `image`, we can use the following API's to access the image scales and manage scales.
+```{note}
+We will use the image object as context in the following examples.
+```
+
+## Default scales
+
+Plone allows you in the `/@@imaging-controlpanel` to configure which scales are available and what dimensions they should have. By default we have the following scales configured:
+
+* large 768:768
+* preview 400:400
+* mini 200:200
+* thumb 128:128
+* tile 64:64
+* icon 32:32
+* listing 16:16
+
+You can add or change scale as you like.
+
+
+## Image resolving by URI
+
+Plone can resolve an image scale via URI in different ways.
+
+### By field and scale name
+
+Given our `news-item1` with the field `image`, we can get the name scale `thumb` as follows:
+
+http://localhost:8080/Plone/news-item1/@@images/image/thumb
+
+To get the original image, you can leave out the scale:
+
+http://localhost:8080/Plone/news-item1/@@images/image
+
+
+
+### By cacheable scale UID name
+
+When an image scale was created, it will be cached under the name `UID.jpeg` in the object annotations and can be resolved by it as follows:
+
+http://localhost:8080/Plone/news-item1/@@images/3d182f34-8773-4f20-a79d-8774c3151b7e.jpeg
+
+This very useful for caching URL's in Varnish or the Browser. In case the scale definitions have changed, they will be saved again under a different UID, so that the URL will change and enforce the Browser or a cache proxy like Varnish to fetch it again.
```

Comment:
```suggestion
This very useful for caching URL's in Varnish or the Browser. In case the uploaded image or scale definitions have changed, they will be saved again under a different UID, so that the URL will change and enforce the Browser or a cache proxy like Varnish to fetch it again.
```

Reply:
most important: new image was uploaded.

Reply:
Incorporated in b6c77816267f8940973a85010a4f37dc9730f58e